genesis/src/lib.rs

```diff
@@ -14,3 +14,19 @@ pub struct Base64Account {
     pub data: String,
     pub executable: bool,
 }
+
+#[derive(Serialize, Deserialize, Debug)]
+pub struct ValidatorAccountsFile {
+    pub validator_accounts: Vec<Base64ValidatorAccount>,
+}
+
+/// A validator account where the data is encoded as a Base64 string.
+/// Includes the vote account and stake account.
+#[derive(Serialize, Deserialize, Debug)]
+pub struct Base64ValidatorAccount {
```

Comment:
I think this struct needs a new name (and comment) now, right? There isn't any base64-encoded data.

Reply:
```suggestion
/// Info needed to create a staked validator account, including relevant balances and vote- and stake-account addresses
#[derive(Serialize, Deserialize, Debug)]
pub struct StakedValidatorInfo {
```
Not totally set on this struct name, if you can think of something better. I originally had `Data` in there, but kind of confusing wrt account data.